src/main/java/org/osiam/addons/administration/controller/LoginController.java

```diff
@@ -31,6 +30,8 @@
     @Inject
     private GeneralSessionData generalSessionData;
 
+    private boolean showAuthCodeErrorMessage = false;
```

Comment:
This will not work as expected. The controller is a stateless singleton bean that you've just introduced state to. Just put the flag in the model, if an error happened.